docs/src/componentDocs/Header/markdown/HeaderAPIDocs.mdx

```diff
@@ -1 +1,135 @@
-### Header API Docs
+import Box from '@mui/material/Box';
+import headerSmallImage from '../images/header_small.png';
+import headerLargeImage from '../images/header_large.png';
+
+# Header
+
+The `<Header>` component is used at the top of the page to display page information. It shows a title and has optional parameters to show a subtitle, background image, navigation button, and multiple action buttons. The header can also be configured to expand/collapse as desired. For expand/collapse behavior tied to the screen scroll position, you should use the [`<CollapsibleHeaderLayout>`](./CollapsibleHeaderLayout.md) component.
+
+<Box
+    sx={{
+        display: 'flex',
+        justifyContent: 'space-evenly',
+        alignItems: 'center',
+        mb: 2,
+        width: '100%',
+        textAlign: 'center',
+    }}
+>
+    <Box
+        sx={{
+            width: '40%',
+        }}
+        component="img"
+        alt="Header component"
+        src={headerSmallImage}
+    />
+    <Box
+        sx={{
+            width: '40%',
+        }}
+        component="img"
+        alt="Header component"
+        src={headerLargeImage}
+    />
+</Box>
+
+## API
+
+<Box>
+
+| Prop Name           | Description                                                                                                                | Type                                                                             | Required | Default                           |
+| ------------------- | -------------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------| -------- | --------------------------------- |
+| actionItems         | Array of icons / actions to display on the right                                                                           | `Array<HeaderIcon \| HeaderActionComponent>`                                     | no       |                                   |
```

Comment:
Can we provide links to the type definition 
https://github.com/etn-ccis/blui-react-native-component-library/blob/6c996dc02c884f60cfdc62938e93836e6d641666/components/src/core/__types__/index.ts#L34

https://github.com/etn-ccis/blui-react-native-component-library/blob/6c996dc02c884f60cfdc62938e93836e6d641666/components/src/core/__types__/index.ts#L42